Demo/Common/IntegrationExamples/Components/CardComponentExample.swift

```diff
@@ -0,0 +1,138 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import AdyenCard
+import AdyenComponents
+import AdyenSession
+
+internal final class CardComponentExample: InitialDataFlowProtocol {
+
+    // MARK: - Properties
+
+    internal var session: AdyenSession?
+    internal var cardComponent: PresentableComponent?
+    internal weak var presenter: PresenterExampleProtocol?
+
+    // MARK: - Initializers
+
+    internal init() {}
+
+    // MARK: - Networking
+
+    internal func requestInitialData(completion: ((PaymentMethods?, Error?) -> Void)?) {
```

Comment:
The completion is not being used anywhere at the moment. What is the reason for it?

Reply:
I am keeping it to add loading spinner on the screen for api response.